.github/workflows/test_build_deploy.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       - name: Test
         run: |
           set -o pipefail
-          yarn test --coverage --coverage-reporters=lcov --detectOpenHandles=false | tee ./unit-tests.log
+          yarn test --coverage --coverage-reporters=lcov --detectOpenHandles=false 2>&1 | tee ./unit-tests.log
```

Comment:
imma leave this here [what does 2>&1 mean](https://stackoverflow.com/questions/818255/what-does-21-mean)